dataplane/standalone/route.h

```diff
@@ -41,4 +41,43 @@ class Route : public APIBase {
   sai_status_t set_attribute(_In_ const sai_attribute_t* attr);
 };
 
+class NextHop : public APIBase {
```

Comment:
NextHop and NHGs are in separate header files in the SAI project. Would be better if they're split out in the same organization so it's more obvious how to reference between the two?

Reply:
done